app/Services/Scanner.php

```diff
@@ -28,6 +28,32 @@ public function laravelVersion( $options )
         return  "11.0.0";
     }
 
+    /**
+     * Detect octane setup and flavor 
+     */
+    public function octaneFlavor( array $options )
+    {
+        // TODO: OCTANE which laravel versions are actually compatible? 11
```

Comment:
Should we even check if the laravel and octane versions are compatible? The user has installed them so why would we not follow their setup?

Reply:
Yuup you've got a point there! I was deliberating whether it's a necessary evil or not.  
Since Octane is only available for later Laravel versions starting from 8, other previous versions might not work with octane at all. Since we already detect the Laravel version of a project, it seemed that including octane detection for those other versions did not make sense. Octane might not even be configured, and in the up chance there's a binary in there, or a laravel/octane in composer, would the Dockerfile created with octane snippets even work? Or will there be dependencies not available in previous Laravel versions. What you think?